src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -234,6 +240,38 @@ const Actions: React.FC<Props> = ({
         };
     };
 
+    const mapsLinkForParcels = (): string => {
+        const mapsLinkList = ["https://www.google.com/maps/dir/"];
+        selectedParcels.map((parcel) => {
+            mapsLinkList.push(parcel.addressPostcode + "/");
+        });
+        mapsLinkList.push("/");
+        const mapsLinkString = mapsLinkList.join("");
+        return mapsLinkString;
```

Comment:
I think there is no need for the mapsLinkList, we could have this as a string and use string.concat rather than list.push, perhaps? Then there is no need to join the list elements :)

Reply:
Yep good plan - I think I'm just more comfortable with array functions than string functions

Reply:
looks good!